wsclient/src/main/java/org/hpccsystems/ws/client/BaseHPCCWsClient.java

```diff
@@ -574,6 +574,10 @@ protected Integer getStubConnectionTO() throws AxisFault
     static public Stub setStubOptions(Stub thestub, Connection connection) throws AxisFault
     {
         Options opt = thestub._getServiceClient().getOptions();
+        EndpointReference toRef = opt.getTo();
+        String toAddress = toRef.getAddress() + (toRef.getAddress().contains("?") ? "&" : "?") + "rawxml_";
```

Comment:
@rpastrana I looked around to see if there was a different way to set a query parameter in Axis2, but wasn't able to find another method. However, I do wonder if we should be using the URL & URI classes here to add the query param instead of doing it manually.

Reply:
I considered URL, but you were in the midst of removing URL from our project. 
At this point, it doesn't seem worthwhile to introduce URL here to add a query param

Reply:
Looked into the URL/URI classes since I'm adding the comment your suggested, but didn't find any convenient methods allowing us to easily add a query param